### ISSUE TITLE
AttributeError for isascii on Ubuntu onionshare v2.5 snap in cli chat mode

### ISSUE THREAD

**Issue by caseyrichins** [onionshare/onionshare#1553]
When attempting to start a chat, the chat starts successfully but after starting the chat and browsing to the onion url, any attempt to type a message or change a username results in a Attribute error on the console of the hosting server and no message is sent in the chat window. 

```message handler error
Traceback (most recent call last):
  File "/snap/onionshare/15/lib/python3.6/site-packages/engineio/server.py", line 622, in _trigger_event
    return self.handlers[event](*args)
  File "/snap/onionshare/15/lib/python3.6/site-packages/socketio/server.py", line 776, in _handle_eio_message
    self._handle_connect(eio_sid, pkt.namespace, pkt.data)
  File "/snap/onionshare/15/lib/python3.6/site-packages/socketio/server.py", line 669, in _handle_connect
    'connect', namespace, sid, self.environ[eio_sid])
  File "/snap/onionshare/15/lib/python3.6/site-packages/socketio/server.py", line 745, in _trigger_event
    return self.handlers[namespace][event](*args)
  File "/snap/onionshare/15/lib/python3.6/site-packages/flask_socketio/__init__.py", line 284, in _handler
    *args)
  File "/snap/onionshare/15/lib/python3.6/site-packages/flask_socketio/__init__.py", line 749, in _handle_event
    ret = handler()
  File "/snap/onionshare/15/lib/python3.6/site-packages/onionshare_cli/web/chat_mode.py", line 123, in server_connect
    if self.validate_username(session.get("name")):
  File "/snap/onionshare/15/lib/python3.6/site-packages/onionshare_cli/web/chat_mode.py", line 54, in validate_username
    and username.isascii()
AttributeError: 'str' object has no attribute 'isascii'
[2022-03-22 14:44:17,177] ERROR in app: Exception on /update-session-username [POST]
Traceback (most recent call last):
  File "/snap/onionshare/15/lib/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/snap/onionshare/15/lib/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/snap/onionshare/15/lib/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/snap/onionshare/15/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/snap/onionshare/15/lib/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/snap/onionshare/15/lib/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/snap/onionshare/15/lib/python3.6/site-packages/onionshare_cli/web/chat_mode.py", line 91, in update_session_username
    if self.validate_username(username):
  File "/snap/onionshare/15/lib/python3.6/site-packages/onionshare_cli/web/chat_mode.py", line 54, in validate_username
    and username.isascii()
AttributeError: 'str' object has no attribute 'isascii'```

**Comment by mig5** [comment]
`.isascii` requires Python 3.7 or higher, and the output shows the user is on Python 3.6. 

I thought we were already hard-requiring Python 3.7 or higher but it seems not, needs fixing.

**Comment by mig5** [comment]
@micahflee do you think this is or can be fixed in the Snap for 2.6.dev1? I'm confused how Python 3.6 was even able to be used here, or is the 3.7 a dependency only in an yet-to-be-released version?

**Comment by mig5** [comment]
I believe this would be resolved in 2.6.1, and we've had no other reports of it. Please reopen if not.
